### DISCUSSION TITLE
zh/guide/markdown/flowchart

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#459] [Announcements]
# zh/guide/markdown/flowchart

让你的 VuePress 站点中的 Markdown 文件支持流程图。 我们使用 flowchart.js 来支持这一功能。

https://theme-hope.vuejs.press/zh/guide/markdown/flowchart.html

**Comment by shirayner** [comment]
希望能支持plantuml
